### PR TITLE
Linked list zip

### PR DESCRIPTION
**Pull request by bashamtg**

### CODE REVIEW

**Comment by bashamtg** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
   - [x] Summary, Description, Approach & Efficiency, Solution
   - [x] Picture of whiteboard
   - [x] Link to code
 - [X] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
   - [X] “Happy Path” - Expected outcome
   - [ ] Expected failure
   - [ ] Edge Case (if applicable/obvious)

File: python/code_challenges/linked_list_zip.py [hunk 1]
```diff
@@ -1,2 +1,21 @@
...
+    while a_node and b_node:
+        # store next pointers
+        a_next = a_node.next
+        b_next = b_node.next
+        # swap
+        b_node.next = a_next
+        a_node.next = b_node
+        # drive
+        a_node = a_next
+        b_node = b_next
```
Comment (jstreifel-33) [inline python/code_challenges/linked_list_zip.py:19]: Looks like the `test_a_shorter` test is failing because you aren't maintaining your reference to b.next upon looping.

If list a is shorter, your function just ends, as it's currently written.
